README.md

```diff
@@ -84,7 +84,9 @@ If your application is running outside of Google Cloud Platform, such as locally
   2. Copy the key somewhere your application can access it. Be sure not to expose the key publicly.
   3. Set the environment variable `GOOGLE_APPLICATION_CREDENTIALS` to the full path to the key. The debug agent will automatically look for this environment variable.
 
-3. Alternatively, if you are running your application on a development machine or test environment where you are using the [`gcloud` command line tools][gcloud-sdk], and are logged using `gcloud beta auth application-default login`, you already have sufficient credentials, and a service account key is not required.
+If you are running your application on a development machine or test environment where you are using the [`gcloud` command line tools][gcloud-sdk], and are logged using `gcloud beta auth application-default login`, you already have sufficient credentials, and a service account key is not required.
+  
+Alternatively, you may set the `keyFilename` or `credentials` configuration field to the full path or contents to the key file, respectively. Setting either of these fields will override both of the above options. (See the [default configuration](config.js) for more details.)
```

Comment:
My intuition is that an env_var change is easier than a code change so it would be nice if `GOOGLE_APPLICATION_CREDENTIALS` overrode `keyFilename` and `credentials`. Maybe we could open an issue and see if auto auth will change their precedence? If not, we should change our stuff to match them for consistency.